ddtrace/profiling/collector/stack.pyx

```diff
@@ -2,7 +2,7 @@
 from __future__ import absolute_import
 
 import sys
-import threading
+import threading as ddtrace_threading
```

Comment:
Do we need this alias? If so we should leave a comment.

Reply:
with the current coding in this module I believe we don't. However it probably makes it clearer here that we are using the copy of the `threading` module that our library ends up using. This is only relevant when module cloning is triggered.